weblate/static/style-bootstrap.css

```diff
@@ -447,6 +451,40 @@ legend {
 .search-group {
     margin: 0 15px 15px 0;
 }
+.query-field {
+    margin-right: 0;
+}
+.search-group input{
+    height: auto;
+}
+.search-group li a {
+    cursor: pointer;
+}
+.search-label svg {
+    height: 16px;
+}
+.query-field .btn.btn-primary{
+    font-size: initial;
+}
+.sort-field {
+    margin-right: 0;
+}
+@media (min-width: 768px) {
+    .sort-field {
+        margin-left: -15px;
+    }
+}
+#query-sort-toggle {
+    width: 40px;
+    padding-left: 0;
+    padding-right: 0;
+}
+#query-sort-toggle span:not(.active) {
+    display: none;
+}
+#query-sort-dropdown {
+    width: 200px;
```

Comment:
Hard-coding width will probably go wrong in some translation. Can this be dynamic?

Reply:
It can be dynamic, but it will constantly resize based on the width of the test in the button. I checks in all different device widths and seems to work fine. I am okay with just keeping it dynamically change to fit the width of the text.